src/platform/silabs/wifi/WifiInterface.h

```diff
@@ -16,7 +16,9 @@
  */
 #pragma once
 
+#include <app-common/zap-generated/cluster-enums.h>
```

Comment:
This include will also be necessary in the other PR (but i think it gets merged in this branch anywais)